packages/init/src/SliceMachineInitProcess.ts

```diff
@@ -137,7 +137,11 @@ export class SliceMachineInitProcess {
 				appName: pkg.name,
 				appVersion: pkg.version,
 			});
-			await setupSentry();
+			try {
+				await setupSentry();
+			} catch (error) {
+				// Noop, we probably couldn't determine the Sentry environment
```

Comment:
Could we add a quick explicit message that we are intentionally not providing feedback or halting the process?

```js
// noop - We don't want to stop the user from using Slice Machine because of
// failed tracking set up. We probably couldn't determine the Sentry
// environment.
```